package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
You need to add the generate schema module to the `dependencies` or this will error on production